.travis.yml

```diff
@@ -0,0 +1,54 @@
+sudo: required
+dist: trusty
```

Comment:
Will you use xenial instead? travis already supports this newer version IIRC.

Reply:
Passes with xenial